scss/_type_matrix.scss

```diff
@@ -0,0 +1,121 @@
+////
+/// @group o-typography
+/// @link http://registry.origami.ft.com/components/o-typography
+////
+
+// Typographic matrix
+
+// Sans
+@mixin oTypographySansSize($level) {
+	@include font-size(get-font-size(sans, $level), get-line-height(sans, $level));
+}
+@mixin oTypographySans($level) {
+	@include oTypographySansSize($level);
+	font-family: $o-typography-sans;
+	font-weight: oFontsWeight(light);
+}
+
+// Sans bold
+@mixin oTypographySansBoldSize($level) {
+	@include font-size(get-font-size(sansBold, $level), get-line-height(sansBold, $level));
```

Comment:
https://github.com/Financial-Times/next-type/blob/master/_mixins.scss#L3
Size is used to only set `font-size` and `line-height`. I imagine there are cases where you only want to change the size without re-declaring font-family etc. I've updated the table in the README with the correct mixins.